main/src/cgeo/geocaching/models/PocketQueryList.java

```diff
@@ -22,6 +22,7 @@
     private final String guid;
     private final int maxCaches;
     private final String name;
+    private boolean downloadable = false;
```

Comment:
I **really** don't like making a previously immutable class mutable. Could you find a way to determine whether the `PocketQueryList` is downloadable at construction time instead?